### PR TITLE
Chore(spec): update test spec to reflect main branch 

### PR DESCRIPTION
**Pull request by hoangdv2429**

### CODE REVIEW

File: tests_spec/eibc.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
-| 5  | Rollapp token ibc-transfer timeout | Same as the case above | 🛑 <br> (missing) | <ul><li> send rollapp IBC denom from dymension to rollapp </li> <li> Same as the case above </li></ul> | Partly solved <br> (lack query `packet commitment` left on the rollapp) |  [ibc_timeout_test](../tests/ibc_timeout_test.go)  |
+| 1  | Rollapp token Demand order is created upon memo submission and fulfilled |At least 2 rollapps running, Rollapp A and B. Rollapp B is at finalized height < Rollapp A finalized height. Rollapp B has a channel-id different from the hub-channel-id. | 🛑 <br> (missing) | Demand order for Rollapp tokens created and fulfilled upon memo submission. | Partly solved <br> (lack query `packet commitment` left on the rollapp) | [TestEIBCFulfillment_EVM](../tests/ibc_grace_period_test.go#28) [TestEIBCFulfillment_Wasm](../tests/eibc_fulfillment_test.go#322) |
```
Comment (trinitys7) [inline tests_spec/eibc.md:5]:
```suggestion
| 1  | Rollapp token Demand order is created upon memo submission and fulfilled |At least 2 rollapps running, Rollapp A and B. Rollapp B is at finalized height < Rollapp A finalized height. Rollapp B has a channel-id different from the hub-channel-id. | ✅ | Demand order for Rollapp tokens created and fulfilled upon memo submission. | ✅ | [TestEIBCFulfillment_EVM](../tests/ibc_grace_period_test.go#28) [TestEIBCFulfillment_Wasm](../tests/eibc_fulfillment_test.go#322) |
```

File: tests_spec/eibc.md [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
+| 2  | DYM Demand order is created upon memo submission and fulfilled | Same as the case above | 🛑 <br> (missing) | Timeout triggered for EIBC transaction from hub to rollapp due to non-finalization. | Partly solved <br> (lack query `packet commitment` left on the rollapp) | [TestEIBCTimeoutHubToRollapp](../tests/eibc_timeout_test.go#24) |
```
Comment (trinitys7) [inline tests_spec/eibc.md:6]:
```suggestion
| 2  | DYM Demand order is created upon memo submission and fulfilled | Same as the case above | ✅ | Timeout triggered for EIBC transaction from hub to rollapp due to non-finalization. | ✅ | [TestEIBCTimeoutHubToRollapp](../tests/eibc_timeout_test.go#24) |
```

File: tests_spec/eibc.md [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
+| 3  | 3rd party token (i.e blumbus) Demand order is created upon memo submission and fulfilled | Same as the case above | 🛑 <br> (missing) | <ul><li> Eibc demand order created</li><li> Fulfill the eibc demand order</li><li> Verify the originator gets the funds minus the fee right away</li><li> Verify the fulfiller gets the full amount after the grace period is finalized</li><li> No packet commitment left on the rollapp (i.e. ack received). Can be queried using the CLI rollapp-evm q ibc channel packet-commitments transfer \<channel-id></li></ul>| 🛑 <br> (missing) | [TestEIBCFulfillment_ThirdParty_EVM](../tests/eibc_fulfillment_test.go#1006) [TestEIBCFulfillment_ThirdParty_Wasm](../tests/eibc_fulfillment_test.go#1405)|
```
Comment (trinitys7) [inline tests_spec/eibc.md:7]:
```suggestion
| 3  | 3rd party token (i.e blumbus) Demand order is created upon memo submission and fulfilled | Same as the case above | ✅ | <ul><li> Eibc demand order created</li><li> Fulfill the eibc demand order</li><li> Verify the originator gets the funds minus the fee right away</li><li> Verify the fulfiller gets the full amount after the grace period is finalized</li><li> No packet commitment left on the rollapp (i.e. ack received). Can be queried using the CLI rollapp-evm q ibc channel packet-commitments transfer \<channel-id></li></ul>| ✅ | [TestEIBCFulfillment_ThirdParty_EVM](../tests/eibc_fulfillment_test.go#1006) [TestEIBCFulfillment_ThirdParty_Wasm](../tests/eibc_fulfillment_test.go#1405)|
```

File: tests_spec/eibc.md [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
+| 4  | DYM ibc transfer timeout | Same as the case above | 🛑 <br> (missing) | <ul><li> send **dym** from dymension to rollapp </li> <li>set minimum packet timeout (i.e. 1ms) so that the packet is not relayed before it is picked by a relayer  </li><li> sender should get funds back after grace period. </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id></li> <li> demand order is created with correct ibc denom and fee </li> <li>demand order can get fulfilled </li> <li>upon timeout the funds should go to the fulfiller </li> </ul>| Partly solved <br> (lack query `packet commitment` left on the rollapp) | TODO |
```
Comment (trinitys7) [inline tests_spec/eibc.md:8]: Why we make this one TODO, I thought we've already do it on timeout test?

File: tests_spec/eibc.md [hunk 5]
```diff
@@ -2,21 +2,21 @@
...
+| 5  | Rollapp token ibc-transfer timeout | Same as the case above | 🛑 <br> (missing) | <ul><li> send rollapp IBC denom from dymension to rollapp </li> <li> Same as the case above </li></ul> | Partly solved <br> (lack query `packet commitment` left on the rollapp) |  TODO  |
```
Comment (trinitys7) [inline tests_spec/eibc.md:9]: Same, why we make this one TODO?

File: tests_spec/eibc.md [hunk 6]
```diff
@@ -2,21 +2,21 @@
...
+| 18 | Demand order fulfillment in 2 different rollapps |  Same as the case above | 🛑 <br> (missing) | <ul> <li> fulfill the order coming from rollapp 1 </li> <li> fulfill the order coming from rollapp 2 </li> <li>user gets amount minus fee for demand order coming from rollapp 1 </li> <li> market maker gets amount upon grace period of IBC packet coming from rollapp 1 </li>  <li>user gets amount minus fee for demand order coming from rollapp 2 </li> <li> market maker gets amount upon grace period of IBC packet coming from rollapp 2 </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id></li> </ul> | 🛑 <br> (missing) | [TestEIBCFulfillment_two_rollapps](../tests/eibc_fulfillment_test.go#625) |
```
Comment (trinitys7) [inline tests_spec/eibc.md:22]:
```suggestion
| 18 | Demand order fulfillment in 2 different rollapps |  Same as the case above | ✅ | <ul> <li> fulfill the order coming from rollapp 1 </li> <li> fulfill the order coming from rollapp 2 </li> <li>user gets amount minus fee for demand order coming from rollapp 1 </li> <li> market maker gets amount upon grace period of IBC packet coming from rollapp 1 </li>  <li>user gets amount minus fee for demand order coming from rollapp 2 </li> <li> market maker gets amount upon grace period of IBC packet coming from rollapp 2 </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id></li> </ul> | ✅ | [TestEIBCFulfillment_two_rollapps](../tests/eibc_fulfillment_test.go#625) |
```

File: tests_spec/rollapp_genesis.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
...
-| 1  | Rollapp genesis tokens on the hub upon channel creation | <ul> <li> Create relevant genesis accounts and metadata info </li> <li> Upon channel creation, trigger genesis events in both rollapp and hub (special tx)  </li> </ul> |  🛑 <br> (missing) | <ul> <li> denometadata should be created </li> <li> VFC should be created on hub (i.e erc20) </li> <li> Rollapp tokens with relevant amount should be locked on the rollapp </li> <li> genesis accounts should have the relevant balance on the hub </li> </ul> | 🛑 <br> (missing) | TODO |
+| 1  | Rollapp genesis tokens on the hub upon channel creation | <ul> <li> Create relevant genesis accounts and metadata info </li> <li> Upon channel creation, trigger genesis events in both rollapp and hub (special tx)  </li> </ul> |  🛑 <br> (missing) | <ul> <li> denometadata should be created </li> <li> VFC should be created on hub (i.e erc20) </li> <li> Rollapp tokens with relevant amount should be locked on the rollapp </li> <li> genesis accounts should have the relevant balance on the hub </li> </ul> | ✅ | [TestRollappGenesisEvent_EVM](../tests/rollapp_genesis_event_test.go#24) |
```
Comment (trinitys7) [inline tests_spec/rollapp_genesis.md:5]:
```suggestion
| 1  | Rollapp genesis tokens on the hub upon channel creation | <ul> <li> Create relevant genesis accounts and metadata info </li> <li> Upon channel creation, trigger genesis events in both rollapp and hub (special tx)  </li> </ul> |  ✅ | <ul> <li> denometadata should be created </li> <li> VFC should be created on hub (i.e erc20) </li> <li> Rollapp tokens with relevant amount should be locked on the rollapp </li> <li> genesis accounts should have the relevant balance on the hub </li> </ul> | ✅ | [TestRollappGenesisEvent_EVM](../tests/rollapp_genesis_event_test.go#24) |
```

**Review by trinitys7 — CHANGES_REQUESTED** [review]

File: tests_spec/eibc.md (hunk 4 above)
Comment (hoangdv2429) [reply tests_spec/eibc.md:8]: Nice catch. Didn't see we cover 2 cases there, updated

**Review by trinitys7 — APPROVED** [review]

**Pull request merged**